src/components/index.js

```diff
@@ -1,4 +1,5 @@
 export * from './NavigationBar'
 export * from './Signup'
 export * from "./Layout";
-export * from './Inputs';
\ No newline at end of file
+export * from './inputs';
```

Comment:
name should be starts with upper case, change folder name to Inputs

Reply:
I did now, but cannot commit it as git doesn't recognize it as a change.